mysqltuner.pl

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/perl
-# mysqltuner.pl - Version 2.5.2
+#!env perl
```

Comment:
removing the full path breaks the shebang when using bash

```
$ ./mysqltuner.pl --help
-bash: ./mysqltuner.pl: env: bad interpreter: No such file or directory

$ bash --version
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

$ which env
/usr/bin/env
```

Reply:
why do we make such change?

Reply:
- https://github.com/major/MySQLTuner-perl/pull/767